client-v2/src/shared/components/input/InputImage.tsx

```diff
@@ -142,6 +142,9 @@ export interface InputImageContainerProps {
   defaultImageUrl?: string;
   useDefault?: boolean;
   message?: string;
+  replaceImage: boolean;
+  setDisplayImageReplaceToggle: (display: boolean) => void;
```

Comment:
This couples our image input to the idea of an `imageReplace` field or similar, which future image fields or usages of this component (I'm thinking editions, although there might be other cases) may not need. Could we move this code outside of this component and deal with this logic in the `onChange` handler instead? Happy to talk through or pair on this if needed 👍

Reply:
Hmm interested by your suggestion. The problem we found is that the imageInput needs to pass certain temporary information (has this image been selected) up to its parent (ArticleParentFormInline - where the toggle is) and this appears to be simplest way to do it. 

Do you mean adding this state into the store in `ImageInput` and then pulling it off in `ArticleParentFormInline`?
I guess we should be cautious about coding around future problems that don't currently exist?